fiverr/passable
# [REFACTOR] passableResult: remove flow, add unit tests, align jsdoc

| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Deprecations?    | no
| Documentation?   | no
| Tests added?     | yes

- Check serialisation of passable result (added a unit tests).
- Remove flow from passable result file.
- Align some jsdoc in passable result file.
- Remove some unused code in passable result file.
